src/backend/services/request_validators.py

```diff
@@ -396,3 +396,30 @@ async def validate_organization_request(session: DBSessionDep, request: Request)
         raise HTTPException(
             status_code=400, detail=f"Organization with name: {name} already exists."
         )
+
+
+async def validate_user_request(session: DBSessionDep, request: Request):
```

Comment:
If this is to explicitly check create User requests, let's change the method  name to `validate_create_user_request`

Reply:
Done